server/src/schemas/usersSchema.ts

```diff
@@ -0,0 +1,73 @@
+import { gql } from "apollo-server-express";
+
+export const UsersTypeDefs = gql`
+  type User {
+    id: ID!
+    first_name: String!
+    last_name: String!
+    email: String!
+    type: Type!
+    privilege: Privilege!
+    registration_date: Date!
+    balance: Int!
```

Comment:
I thought we weren't doing user balances?

Reply:
Forgot tbh